pandas-iris/{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -3,9 +3,6 @@ package_name = "{{ cookiecutter.python_package }}"
 project_name = "{{ cookiecutter.project_name }}"
 kedro_init_version = "{{ cookiecutter.kedro_version }}"
 
-[tool.isort]
-profile = "black"
-
```

Comment:
Thanks, I notice I haven't removed this from the `kedro` branch, I will make a clean up PR similar to this shortly.

Reply:
I removed it in https://github.com/kedro-org/kedro/pull/3072/

Reply:
It's on develop though

Reply:
I see, i will leave it then since it's just clean up and we will merge it back to main eventually.